src/main/java/seedu/inventory/logic/commands/ClearCommand.java

```diff
@@ -13,14 +14,15 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "Inventory list has been cleared!";
+    public static final String MESSAGE_SUCCESS = "Inventory, staff, sale orders and purchase orders have been cleared!";
```

Comment:
The message here is a bit incorrect for staff list. When the clear command is executed, staff list will be reset to its default state.

Reply:
Done, made the change